src/main/java/knox/spring/data/neo4j/controller/KnoxController.java

```diff
@@ -277,7 +277,7 @@ public KnoxController(DesignSpaceService designSpaceService) {
     }
 	
 	/**
-	 * @api {post} /branch/repeat Repeat
+	 * @api {post} /branch/Repeat
```

Comment:
why did this get capitalized?

Reply:
I took a look and found out the first line of the comments for operator endpoints are different from those for the history endpoints. I would correct this.